projects/cdk/components/dialog-host/dialog-host.component.ts

```diff
@@ -23,7 +31,40 @@ export class TuiDialogHostComponent<T extends TuiAriaDialogContext> {
     );
 
     constructor(
-        @Inject(TUI_DIALOGS)
-        private readonly dialogsByType: Observable<readonly T[]>[],
+        @Inject(TUI_DIALOGS) private readonly dialogsByType: Observable<readonly T[]>[],
+        @Inject(WINDOW) private readonly windowRef: Window,
```

Comment:
I think it would be a good idea to add HISTORY token to @ng-web-apis/common

Reply:
done